Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -28,20 +29,517 @@ static void Main(string[] args)
+                                temp_list.Clear();
+                                temp_list = new List<int>(new_temp_list);
+                                new_temp_list.Clear();
+                                if (temp_list.Count() == 1)
+                                {
+                                    
+                                    break;
+                                }
+
+                            }
+                            break;
+
+                        case 't':
+                            if (temp_list.Count == 0)
+                            {
+                                
+                                int min_cal = cal.Min();
+
+                                for (int i = 0; i < cal.Length; i++)
+                                {
+                                    if (cal[i] == min_cal)
+                                    {
+                                        temp_list.Add(i);
+                                    }
+                                }
+
+                               
+
+                                if (temp_list.Count() == 1)
+                                {
+
+                                    
+                                    break;
+                                }
+                            }
+                            else if (temp_list.Count > 1)
+                            {
+                                int temp_var = 999;
+                                
+
+                                foreach (int idx in temp_list)
+                                {
+
+                                    if (temp_var > cal[idx])
+                                    {
+                                        temp_var = cal[idx];
+                                        new_temp_list.Clear();
+                                        new_temp_list.Add(idx);
+
+                                    }
+                                    else if (temp_var == cal[idx]) 
+                                    {
+                                        new_temp_list.Add(idx);
+                                    }
+                                }
+                                temp_list.Clear();
+                                temp_list = new List<int>(new_temp_list);
+                                new_temp_list.Clear();
+                                new_temp_list.Clear();
+                                if (temp_list.Count() == 1)
+                                {
+                                   
+                                    break;
+                                }
+
+                            }
+
+                            break;
+
+
+
+
+
+
+                    }
+
+                    
+                }
+
+                Answer[count - 1] = temp_list[0];
+
+                temp_list.Clear();
+                new_temp_list.Clear();
+            }
+
+            return Answer;   
+
+            throw new NotImplementedException();

```

Comment:
Again we should not throw this once the code is implemented

Reply:
okay